### PR TITLE
Expose Custom events API

### PR DESCRIPTION
**Pull request by imvanzen**
- Add track event method
- Update docs
- Update test

### CODE REVIEW

File: lib/voucherify.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@
 require 'voucherify/service/validation_rules'
 require 'voucherify/service/validations'
 require 'voucherify/service/vouchers'
+require 'voucherify/service/events'
```
Comment (lkulig) [inline lib/voucherify.rb]: let's keep those ordered

File: lib/voucherify/service/events.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'uri'
```
Comment (lkulig) [inline lib/voucherify/service/events.rb]: unused require

File: spec/custom_events_spec.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+require 'rest-client'
+require 'date'
+
+describe 'Custom Events API' do
```
Comment (lkulig) [inline spec/custom_events_spec.rb]: just `Events API` also please rename the file to `events_spec.rb`

File: spec/custom_events_spec.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+require 'rest-client'
+require 'date'
```
Comment (lkulig) [inline spec/custom_events_spec.rb]: unused

File: README.md [hunk 5]
```diff
@@ -46,6 +46,8 @@ API:
 |
 <a href="#segments-api">Segments</a>
 |
+<a href="#custom-events-api">Custom events</a>
```
Comment (lkulig) [inline README.md]: just `Events`

File: README.md [hunk 6]
```diff
@@ -384,6 +386,18 @@ voucherify.segments.delete(id);
 
 ---
 
+### Custom Events API
```
Comment (lkulig) [inline README.md]: just `Events`

File: README.md [hunk 7]
```diff
@@ -384,6 +386,18 @@ voucherify.segments.delete(id);
...
+Methods are provided within `voucherify.events.*` namespace.
+
+- [Custom events](#custom-events)
```
Comment (lkulig) [inline README.md]: this should rather be `[Create Event] (#create-event)` and the following `#### [Create custom event]` changed to `#### [Create Event]` and lastly the link to docs from `[Custom Events]` to `[Create Event]`

**Comment by imvanzen** [comment]
@lkulig Thank you for review. I have solved all reported issues. Please revalidate this PR.

**Pull request merged**